docs/Operators.md

```diff
@@ -485,6 +486,172 @@ expect(node, inputs=[x, y], outputs=[x + y], name="test_add_uint8")
 </details>
 
 
+### <a name="AffineGrid"></a><a name="affinegrid">**AffineGrid**</a>
+
+  Generates a 2D or 3D flow field (sampling grid), given a batch of affine matrices theta
+  (https://pytorch.org/docs/stable/generated/torch.nn.functional.affine_grid.html).
+  An affine matrix `theta` is applied to a position tensor represented in its homogeneous expression. Here is an example in 3D:
+  ```
+  [r00, r01, r02, t0]   [x]   [x']
+  [r10, r11, r12, t1] * [y] = [y']
+  [r20, r21, r22, t2]   [z]   [z']
+  [0,   0,   0,   1 ]   [1]   [1 ]
+  ```
+  where (x, y, z) is the position in the original space, (x', y', z') is the position in the output space.
```

Comment:
```suggestion
  where `(x, y, z)` is the position in the original space, `(x', y', z')` is the position in the output space.
```

style all of these and below as code for readability?

Reply:
document corrected